pom.xml

```diff
@@ -209,6 +214,7 @@
         <dep.parquet.version>1.14.3</dep.parquet.version>
         <dep.plugin.failsafe.version>${dep.plugin.surefire.version}</dep.plugin.failsafe.version>
         <dep.protobuf.version>3.25.5</dep.protobuf.version>
+        <dep.ranger.version>2.5.0</dep.ranger.version>
```

Comment:
Please put this and other version properties that are only ranger specific to plugin/trino-apache-ranger/pom.xml